CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 11.3.18 (07/26/23)
+
+* Fixed enhanced recording of missing `session.command` events when PAM enabled. [#29030](https://github.com/gravitational/teleport/issues/29030) [#29582](https://github.com/gravitational/teleport/pull/29582)
```

Comment:
Optional: I'd consider trimming the changelog down a bit to user-focussed changes. Things like changing the Go version are possibly a bit developer centric, but it's not enough for me to block the merge.

Reply:
I included that because it was included in previous releases. I suspect so that people who track vulnerabilities can know we're building with a version without a vulnerability?